tests/fixtures/workflows/pdr_parse_ingest.json

```diff
@@ -0,0 +1,55 @@
+{
+    "DiscoverPdrs": {
```

Comment:
Minor style thing but elsewhere in this repo we're using 2 spaces for indentation so preference would be to be consistent with 2 spaces.